bin/adkg-cli/README.md

```diff
@@ -114,27 +141,38 @@ adkg-cli run                                \
 
 Notice that we also include two output files used to store the private and public output of the ADKG.
 
-We also include a third file, with `--transcript-out`, which is used to store an encrypted transcript of messages sent during the ADKG.
+We also include a third file, with `--transcript-out`, which is used to store an encrypted transcript of messages sent
+during the ADKG.
 The transcript should be sent to the other participants should a node fail to run the `ADKG` ceremony.
 
 ### Additional options
+
 **Logging**\
 The log level can be changed using the `LOG_LEVEL` env variable.
-The specified level should follow the [`EnvFilter`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives) syntax.
+The specified level should follow the [
+`EnvFilter`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives)
```

Comment:
This is not the case right now but I think there's a risk some markdown parsers might complain about that. We could use [Reference style syntax](https://gist.github.com/roshith-balendran/d50b32f8f7d900c34a7dc00766bcfb9c#reference-style) to take the URL out of line